lib/Raven/Breadcrumbs/ErrorHandler.php

```diff
@@ -41,7 +41,11 @@ public function handleError($code, $message, $file = '', $line = 0, $context = a
 
     public function install()
     {
-        $this->existingHandler = set_error_handler(array($this, 'handleError'), E_ALL);
+        $this->existingHandler = set_error_handler(
```

Comment:
There is no valid reason to wrap a function call in a lambda as you're doing. PHP 5.5 or PHP 4 have nothing to do with this argument. A [callable](http://php.net/manual/en/language.types.callable.php) in PHP is represented in various forms, one of which is an array.